app/styles/pages/public-event.scss

```diff
@@ -64,39 +64,21 @@
           }
 
           .event.name {
-            font-size: 4.5vh;
-            margin-bottom: .5vh;
+            font-size: 2em;
+            margin-bottom: 0;
             margin-top: 0;
           }
 
           .event.time {
-            font-size: 2.25vh;
+            font-size: 1.2em;
```

Comment:
We can use rem instead of em. Since when it comes to nesting of elements, we dont get distorted UI. Here's a good article showing why to use rem for font size.
https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984
Let me know if there is specific reason why you kept em.